docs/application/dotnet/guides/tizen.circularUI/CircleSurfaceEffectBehavior.md

```diff
@@ -0,0 +1,41 @@
+---
+uid: Tizen.Wearable.CircularUI.doc.CircleSurfaceEffectBehavior
+summary: CircleSurfaceEffectBehavior guide
+---
+
+# CircleSurfaceEffectBehavior
+The `CircleSurfaceEffectBehavior` is an effect, which allows you to attach the `CircleSurface` to the [Xamarin.Forms.Page](https://developer.xamarin.com/api/type/Xamarin.Forms.Page/). [CircleSurface](https://developer.tizen.org/development/guides/native-application/user-interface/efl/ui-components/wearable-ui-components/circle-surface) is a Tizen specific component that manages [circle objects](https://developer.tizen.org/development/guides/native-application/user-interface/efl/ui-components/wearable-ui-components/circle-object).
+If you need to add `circle control`(such as [CircleListView](xref:Tizen.Wearable.CircularUI.doc.CircleListView), [CircleDateTimeSelector](xref:Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector), [CircleScrollView](xref:Tizen.Wearable.CircularUI.doc.CircleScrollView), [CircleStepper](xref:Tizen.Wearable.CircularUI.doc.CircleStepper)) in the `Page` in Tizen Xamarin Forms application, there are two ways. One is inserting these controls to `CirclePage`, the other is adding `CircleSurfaceEffectBehavior` in the `Page` that includes `circle control`.
+
+## To add CircleSurfaceEffectBehavior in the Page
+Add `CircleSurfaceEffectBehavior` in the `<Page.Behaviors>`. And then set `RotaryFocusTargetName` property with `circle control` name. `RotaryFocusTargetName` sets the currently focused control that is handled by rotating and display the focused control's circle object. Please refer to below sample code.
+
+_The code example of this guide uses WearableUIGallery's TCListAppender code at the test\WearableUIGallery\WearableUIGallery\TC\TCListAppender.xaml_
+
+For more information, see below links.
+- [CircleSurfaceEffectBehavior API reference](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleSurfaceEffectBehavior.html)
+- [Xamarin.Forms.Page guide](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/controls/pages)
+- [Consuming a Xamarin.Forms Behavior](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/behaviors/creating#consuming-a-xamarinforms-behavior)
+
```

Comment:
Please add a sentence:
Following is the sample code:

Reply:
I change the order of paragraphs according to the other guides.